app/src/modules/content/components/version-menu.vue

```diff
@@ -77,6 +78,24 @@ function useCreateDialog() {
 	const newVersionKey = ref<string | null>(null);
 	const newVersionName = ref<string | null>(null);
 
+	watch(
+		newVersionName,
+		(newName, oldName) => {
+			if (
+				newName === null ||
+				newVersionKey.value ===
+					slugify(oldName ?? '', {
+						separator: '_',
+					})
+			) {
+				newVersionKey.value = slugify(newName ?? '', {
+					separator: '_',
+				});
+			}
+		},
+		{ immediate: true },
+	);
+
```

Comment:
```suggestion
	watch(
		newVersionName,
		(newName, oldName) => {
			if (
				newName === null ||
				newVersionKey.value ===
					slugify(oldName ?? '', {
						separator: '-',
					})
			) {
				newVersionKey.value = slugify(newName ?? '', {
					separator: '-',
				});
			}
		},
		{ immediate: true },
	);

```
In the current app when typing into the key field and pressing "space" it inserts a `-` rather than a `_`